docs/CaliptraIntegrationSpecification.md

```diff
@@ -594,23 +594,31 @@ The following table describes SoC integration requirements.
 | Mailbox                          | SoC shall provide an access path between the mailbox and the application CPU complex on SoCs with such complexes (for example, Host CPUs and Smart NICs). See the [Sender Protocol](#sender-protocol) section for details about error conditions.             | Statement of conformance | Required for Project Kirkland and TDISP TSM       |
 | Fuses                            | SoC shall burn non-field fuses during manufacturing. Required vs. optional fuses are listed in the architectural specification.                                                                                                           | Test on silicon          | Required for UDS threat model                     |
 | Fuses                            | SoC shall expose an interface for burning field fuses. Protection of this interface is the SoC vendor’s responsibility.                                                                                                                   | Test on silicon          | Required for Field Entropy                        |
 | Fuses                            | SoC shall write fuse registers and fuse done via immutable logic or ROM code.                                                                                                                                                             | Statement of conformance | Required for Caliptra threat model                |
+| Fuses                            | SoC shall expose an API for programming Field Entropy as described in the architecture documentation. SoC shall ensure that Field Entropy can only be programmed via this API and shall explicitly prohibit burning of discrete Field Entropy bits and re-burning of already burned Field Entropy entries. | Test on silicon          | Required for Field Entropy                        |
+| Fuses                            | SoC shall ensure that any debug read paths for fuses are disabled in PRODUCTION lifecycle state.| Test on silicon          | Required for Field Entropy                        |
+| Fuses                            | SoC shall ensure that UDS_SEED and Field Entropy supplied to Caliptra come directly from OTP fuses and there are no debug paths to inject new values.| Statement of conformance          | Required for Caliptra Threat model                        |
+| Fuses                            | SoC shall add integrity checks for Caliptra fuses as per SoC policy. | Statement of conformance | Reliability |
+| Fuses                            | SoC should apply shielding/obfuscation measures to protect fuse macro. | Statement of conformance | Required for Caliptra Threat model |
```

Comment:
What type of shielding/obfuscation are we requiring here?  Do we expect all to be obfuscated (SVN, public key hash, other public data, etc.)?

Reply:
This is an optional requirement (hence the term "should" instead of "shall") derived from following line in the main spec. It is difficult to shield fuse macro against physical attacks and we are leaving exact obfuscation mechanism up to the integrator.

"Decapping and fibbing attacks should at least penetrate enough layers and metal shielding to render the part useless, if not being outright impossible to carry out."

Reply:
Understood.  Thank you!